xml/bp_troubleshooting_cephfs.xml

```diff
@@ -87,132 +87,6 @@ ceph daemon mds.<replaceable>NAME</replaceable> dump_ops_in_flight
      </para>
    </sect2>
  </sect1>
- <sect1 xml:id="bp-troubleshooting-cephfuse-debugging">
-   <title>Ceph-Fuse Debugging</title>
```

Comment:
I know I suggested dropping the reference to ceph-fuse, but I would feed more comfortable if @jan--f would confirm that's OK.

Reply:
Since we don't support the fuse client, dropping this makes sense. Please go ahead